sdk/identity/Azure.Identity/src/ScopeUtilities.cs

```diff
@@ -5,13 +5,17 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 
 namespace Azure.Identity
 {
     internal static class ScopeUtilities
     {
         private const string DefaultSuffix = "/.default";
+        private const string ScopePattern = "^[0-9a-zA-Z-.:/]+$";
+
+        private const string InvalidScopeMessage = "The specified scope is not in expected format. Only alphanumeric characters, '.', '-', ':', and '/' are allowed";
```

Comment:
"... characters '.', ..." (no comma after "characters").

Reply:
I think the punctuation here is correct. "alphanumeric characters" is the first element of the list of character / character groups